module/documents/token.mjs

```diff
@@ -127,6 +127,7 @@ export default class TokenDocument5e extends SystemFlagsMixin(TokenDocument) {
       };
       actionConfig.getCostFunction = (...args) => this.getMovementActionCostFunction(type, ...args);
     }
+    CONFIG.Token.movement.actions.crawl.getCostFunction = () => (cost, from, to, distance) => cost + distance;
```

Comment:
Core uses `cost * 2`, which makes the cost of crawling 4 feet instead of 3 feet per foot.